client/js/components/user/DpUserList/DpUserList.vue

```diff
@@ -223,17 +223,22 @@ export default {
     }, 500),
 
     getItemsByPage (page) {
-      const search = {
-        value: this.searchValue
-      }
       this.isLoading = true
       page = page || this.currentPage
 
       this.userList({
         page: {
           number: page ?? 1
         },
-        search: (this.searchValue !== '') ? search : {},
+        filter: {
+          firstnameFilter: {
+            condition: {
+              path: 'firstname',
```

Comment:
do we only want to search for the firstname? shouldn't this be a combination of first- and lastname?

Reply:
yes, it works now! Thank you for the info: done here https://github.com/demos-europe/demosplan-core/pull/3360/commits/7fd0b95adf7c85f92c8465972e9eea909ed03c3b